src/cloud/notifications/account-settings-notifications.ts

```diff
@@ -1,7 +0,0 @@
-import {Notification} from 'src/types'
-import {defaultErrorNotification} from 'src/shared/copy/notifications'
-
```

Comment:
This toast notification isn't needed anymore. Dispatching a notification to the user was less desirable (since the user can't do anything with a bad /orgs/allowances/create response), and we now have separate error handling (honeybadger) in the thunk.